extensions/smallrye-openapi/deployment/pom.xml

```diff
@@ -69,7 +69,7 @@
     <!-- Test -->
     <dependency>
       <groupId>io.quarkus</groupId>
-      <artifactId>quarkus-resteasy-deployment</artifactId>
+      <artifactId>quarkus-resteasy-reactive-deployment</artifactId>
```

Comment:
Is smallrye-openapi supposed to have such a hard dependency?

Reply:
There's no hard dependency, it's `<scope>test</scope>`. Here I'm merely migrating the test suite to use resteasy-reactive instead of classic.

Reply:
Ah okay, it was not clear from the Github UI